src/main/scala/org/neo4j/spark/dataframe/Neo4jDataFrame.scala

```diff
@@ -21,7 +22,9 @@ object Neo4jDataFrame {
                     source: (String, Seq[String]),
                     relationship: (String, Seq[String]),
                     target: (String, Seq[String]),
-                    renamedColumns: Map[String, String] = Map.empty): Unit = {
+                    renamedColumns: Map[String, String] = Map.empty,
+                    partitions: Int = 1,
+                    unwindBatchSize: Int = 10000): Unit = {
     dataFrame.cache()
```

Comment:
should this by default? isn't that something that the user wants to control?

Reply:
ok